README.md

```diff
@@ -71,6 +71,21 @@ export class AppComponent {
 }
 ```
 
+Here's an example of initializing the SDK in a Vue app.
+
+```javascript
+import Vue from "vue";
```

Comment:
How would you feel about using single quotes rather than double quotes in your example? The other examples use single quotes.

Reply:
Fine with that! Pushed a new commit to follow the existing examples. Something we can also change is the wording a bit that way "Here's an example of initializ..." is not repetitive. 


### Initialization Examples with some popular JavaScript libraries and frameworks 

#### [React](https://reactjs.org/) :

```JSX
...
```

#### [Angular](https://angular.io/) with `devMode` enabled to stop recording work in progress :

```javascript
...
```

#### [Vue](https://vuejs.org/) :

```javascript
...
```

![image](https://user-images.githubusercontent.com/463779/80517251-65febd00-894a-11ea-900c-b1710f6e259e.png)

Reply:
Great suggestion! I'll fix that up in a separate PR.